plugins/+ciplugins/+github/BuildSummaryPlugin.m

```diff
@@ -2,27 +2,46 @@
 
 %   Copyright 2024 The MathWorks, Inc.
 
-    methods (Access=protected)
+    properties
+        TaskDetails = {};
```

Comment:
```suggestion
        TaskDetails
```

No need for the default value here

Reply:
I think the default value is needed here to make sure TaskDetails is a cell array and not a struct array. A cell array is required since each element of this array is a struct which can have different number of fields (6 fields for skipped tasks and 5 for others)

Reply:
Makes sense. Thanks.